styles/blocks/02-subtitle.json

```diff
@@ -0,0 +1,13 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/theme.json",
+	"version": 3,
+	"title": "Subtitle",
+	"slug": "text-subtitle",
+	"blockTypes": ["core/heading", "core/paragraph"],
```

Comment:
We should probably also add `core/post-title` to this one and the "title", so we can use it in the context of the site editor. We could also consider the site title.

Reply:
Yes, I believe that once this is merged, we could:

- Modify the `XXL` with the size of the H1.
- Modify the `XL` with the size of the H2.

The old XXL and XL will live on these new text styles. And then we can adjust [this one](https://github.com/WordPress/twentytwentyfive/pull/286) to use the modified XL and XL, instead of the custom variables.